### PR TITLE
BufferGeometry compression example improved and codes optimized

### PR DESCRIPTION
**Pull request by LeonYuanYao**
Sorry for the late commits! I've been busy since chinese new year and the 2019-nCov has just impacted too much... Anyway I just finished codes optimization and improvement, considered many nice advices from others. Hope you accept it.

### CODE REVIEW

**Comment by Mugen87** [comment]
Please avoid any modifications of build and core files.

**Comment by LeonYuanYao** [comment]
> Please avoid any modifications of build and core files.

Should I remove the codes in 'ShaderChunk.js' and accociated glsl files?

**Comment by Mugen87** [comment]
Yes, I think the changes should focus on `webgl_buffergeometry_compression` and `GeometryCompressionUtils`.

**Comment by Mugen87** [comment]
You still have build files included in your PR.

File: examples/jsm/utils/GeometryCompressionUtils.js [hunk 1]
```diff
@@ -28,110 +28,117 @@ var GeometryCompressionUtils = {
 		 * @param {String} encodeMethod		"DEFAULT" || "OCT1Byte" || "OCT2Byte" || "ANGLES"
 		 *
 		 */
-	compressNormals: function ( mesh, encodeMethod ) {
```
Comment (mrdoob) [inline examples/jsm/utils/GeometryCompressionUtils.js]: Do you mind reverting the code style changes?

**Comment by LeonYuanYao** [comment]
Formatting style should be ok now.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**